specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations-authoring.json

```
@@ -0,0 +1,2703 @@
+        "runVerification": {
+          "description": "Represents the option to run verifications. By default, verifications are turned on.",
+          "type": "boolean"
+        }
+      }
+    },
+    "TrainingJobResultObject": {
+      "required": [
+        "result",
+        "jobId",
+        "createdDateTime",
+        "lastUpdatedDateTime",
+        "expirationDateTime",
+        "status",
+        "errors"
+      ],
+      "type": "object",
+      "properties": {
+        "result": {
+          "description": "Represents Training Tasks Detailed Result.",
+          "$ref": "#/definitions/TrainingJobTaskResultObject"
+        },
+        "jobId": {
+          "description": "Gets or sets the jobId.",
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the created time.",
+          "type": "string"
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the last updated time.",
+          "type": "string"
+        },
+        "expirationDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the expiration time.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Gets or sets the job status.",
+          "enum": [
+            "unknown",
+            "notStarted",
+            "running",
+            "succeeded",
+            "failed",
+            "cancelled",
+            "cancelling",
+            "partiallyCompleted"
+          ],
+          "type": "string"
+        },
+        "errors": {
+          "description": "Gets or sets the errors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobErrorObject"
+          }
+        }
+      }
+    },
+    "TrainingJobTaskResultObject": {
+      "required": [
+        "trainedModelLabel",
+        "trainStatus"
+      ],
+      "type": "object",
+      "properties": {
+        "trainedModelLabel": {
+          "description": "Represents trained model label.",
+          "type": "string"
+        },
+        "trainStatus": {
+          "description": "Represents model train status.",
+          "$ref": "#/definitions/TrainingJobStatusResultStatusObject"
+        },
+        "evaluationStatus": {
+          "description": "Represents model evaluation status.",
+          "$ref": "#/definitions/TrainingJobStatusResultStatusObject"
+        }
+      }
+    },
+    "TrainingJobStatusResultStatusObject": {
+      "required": [
+        "percentComplete",
+        "elapsedTime"
+      ],
+      "type": "object",
+      "properties": {
+        "percentComplete": {
+          "format": "int32",
+          "description": "Represents progress percentage.",
+          "type": "integer"
+        },
+        "elapsedTime": {
+          "description": "Represents the elapsed time.\n            This string is in the format hh:mm:ss",
```

Currently, formatting won't carry through so this will look funny. I recommend just:
```suggestion
          "description": "Represents the elapsed time. This string is in the format hh:mm:ss",
```